swan-lake/integration-tools/health-tool.md

```diff
@@ -32,8 +32,9 @@ health:2.0.0 successfully set as the active version.
 
 The Ballerina Health tool supports two main usages as follows.
 
-- [**Package generation:**](#package-generation) generate a Ballerina package from a given FHIR implementation guide.
-- [**Template generation:**](#template-generation) generate Ballerina templates for FHIR APIs from a given FHIR implementation guide.
+- [**FHIR package generation:**](#fhir-package-generation) generate a Ballerina package from a given FHIR implementation guide.
+- [**FHIR template generation:**](#fhir-template-generation) generate Ballerina templates for FHIR APIs from a given FHIR implementation guide.
+- [**CDS template generation:**](#cds-template-generation) generate Ballerina templates for FHIR APIs from a given FHIR implementation guide.
```

Comment:
CDS description is same as FHIR template description

Reply:
Updated: https://github.com/ballerina-platform/ballerina-dev-website/pull/9636/commits/f64bf61fdea84f1436f789c8637eacf16e432475